### ISSUE TITLE
Controlnet

### ISSUE THREAD

**Issue by luck111111111111111** [InstantID/InstantID#233]
In your paper it is mentioned that the input of controlnet is like embeding, why does the input of controlnet have text in the inferfull of reasoning。

**Comment by haofanwang** [comment]
No, there is no text_emb in ControlNet. As said in our paper, we replace text_emb with face_emb.
